casper/src/test/scala/coop/rchain/casper/util/GenesisBuilder.scala

```diff
@@ -51,15 +51,16 @@ object GenesisBuilder {
         proofOfStake = ProofOfStake(
           minimumBond = 0L,
           maximumBond = Long.MaxValue,
-          epochLength = 1,
+          epochLength = 1, // TODO: change to proper quantity
```

Comment:
This is genesis created for testing so it should be fine with fixed value. Does PoS tests depend on _epochLength_ = 1?

Reply:
Since I introduced the block number set capability, I've been making the PoS tests parametric in the epoch length and quarantine length to test the behavior in relation to these quantities, e.g. that a withdrawing validator is not paid before their quarantine period is finished and they are payed at the first epoch change after it's finished.

Reply:
Epoch length of 1 was causing tests with merging block to fail. I've updated your PR to include close block system deploy and default epoch length large enough to prevent conflicts in tests.